examples/dotnet/ContinuousQueryCs/Program.cs

```diff
@@ -1,4 +1,4 @@
-﻿/*
```

Comment:
Were these all BOM issues?

Reply:
Yep, the GitHub review doesn't show the control chars.